Formula/lib/librime.rb

```diff
@@ -0,0 +1,60 @@
+class Librime < Formula
+  desc "Rime Input Method Engine"
+  homepage "https://rime.im"
+  url "https://github.com/rime/librime.git",
+    tag:      "1.12.0",
+    revision: "c7ab6390c143a11d670f3add41218111edb883c9"
+  license "BSD-3-Clause"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "icu4c@76" => :build
+
+  depends_on "boost"
+  depends_on "gflags"
+  depends_on "glog"
+  depends_on "leveldb"
+  depends_on "libtool"
+  depends_on "opencc"
+  depends_on "yaml-cpp"
+
+  resource "marisa" do
+    url "https://github.com/rime/marisa-trie/archive/refs/tags/v0.2.5.tar.gz"
+    sha256 "61d222d335db3e8138a22542b95788f307366986c464a68791d361787f3b9c01"
+  end
+
+  def install
+    resource("marisa").stage do
+      system "autoreconf", "--install", "--verbose"
+      system "./configure", "--disable-silent-rules", *std_configure_args
+      system "make", "install"
+    end
+
+    system "cmake", "-S", ".", "-B", "build",
+                    "-DBUILD_TEST=OFF",
+                    "-DCMAKE_INSTALL_RPATH=#{rpath}",
+                     *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+    # Conflict with opencc library, libmaria.a
+    path = lib/"libmarisa.a"
+    rm path if path.exist?
```

Comment:
Why is is installing static libraries at all? The `std_cmake_args` and `std_configure_args` should tell the build to use shared ones.